static/css/style.css

```diff
@@ -657,6 +657,12 @@ table {
   text-decoration: none !important;
 }
 
+
+.relay-address.alias-copied.alias-copied-fadeout .alias-copied-message {
+  opacity: 0 !important;
```

Comment:
`The !important` may not be required, but I wanted to match previous convention.